ps8/controllers/admin/AdminAjaxPrestashopCheckoutController.php

```diff
@@ -359,7 +359,11 @@ public function ajaxProcessBatchSaveConfiguration()
             $this->validateBatchConfiguration($configuration);
 
             foreach ($configuration as $configurationItem) {
-                $configurationService->set(pSQL($configurationItem['name']), pSQL($configurationItem['value']));
+                if (is_array($configurationItem['value'])) {
+                    $this->setConfiguration($configurationItem['name'], json_encode($configurationItem['value']));
+                } else {
+                    $configurationService->set(pSQL($configurationItem['name']), pSQL($configurationItem['value']));
+                }
```

Comment:
Why use `$this->setConfiguration` instead of `$configurationService->set` ?
Why the received value is not already json serialized by the BO-CDN ?

Reply:
Because of some json payload escaping. 
Specifically " symbol is always escaped before being saved in the configuration, which we have to always clean up before parsing